src/Components/SignUpPage/Register.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { IoMdClose } from "react-icons/io";
+import "./SignUpPage.css";
+
+const Register = ({ handleCloseModal }) => {
+  const [userDetails, setUserDetails] = useState({
+    name: "",
+    email: "",
+    username: "",
+    password1: "",
+    password2: "",
+  });
+
+  const handleChange = (e) => {
+    setUserDetails({ ...userDetails, [e.target.name]: e.target.value });
```

Comment:
Use the updater function

Reply:
I corrected this on the sign-up-page branch already